consensus/consensus.go

```diff
@@ -97,7 +97,7 @@ type Engine interface {
 	// Note: The block header and state database might be updated to reflect any
 	// consensus rules that happen at finalization (e.g. block rewards).
 	FinalizeAndAssemble(chain ChainHeaderReader, header *types.Header, state *state.StateDB, txs []*types.Transaction,
-		uncles []*types.Header, receipts []*types.Receipt, withdrawals []*types.Withdrawal) (*types.Block, error)
+		uncles []*types.Header, receipts []*types.Receipt, withdrawals []*types.Withdrawal, exits []*types.Exit) (*types.Block, error)
```

Comment:
we should think about turning this into some struct at some point.
Might actually be nicer for downstream projects that implement their own consensus engines as well

Reply:
Yeah, we'll also have deposits here for 6110, so I think it's time to figure out a better format for this.

Reply:
maybe time to make a formal BlockBody like in Prysm?

Reply:
It was changed on master branch to use `types.Body`.
All existing tracers in eth/tracers/native have been rewritten to use the new hook system.